lib/getExperiences.js

```diff
@@ -0,0 +1,96 @@
+const jsforce = require('jsforce');
+const { escapeSOQL } = require('escape-soql-and-sosl');
+
+module.exports = function(self, options) {
+ 
+  // Log into salesforce and return a connection object for queries.
+  // TODO: integrate some kind of sensible reuse policy here, without
+  // changing the API, keeping in mind that introduces the need for
+  // retry smarts if the login ends. Connects with the module-level
+  // adminLoginUrl, adminUsername and adminPassword options
+
+  self.connect = async function() {
+    const conn = new jsforce.Connection({
+      loginUrl: self.options.adminLoginUrl || 'https://client-domain.my.salesforce.com'
```

Comment:
Will this fallback help if used? Wouldn't it result in connection errors?

Reply:
I see fallbacks in the login as well. Shouldn't we just stop the show and send a nice error if they're missing?